packages/components/src/List/ListComposition/Toolbar/ListToolbar.component.js

```diff
@@ -18,7 +18,7 @@ const Right = ({ children }) => (
 );
 
 Right.propTypes = {
-	children: PropTypes.oneOfType([PropTypes.element, PropTypes.arrayOf(PropTypes.element)]),
+	children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
```

Comment:
Isn't `PropType.node` sufficient ?

Reply:
It could be but with only node, you can't but an array of children without having a warn